projects/plugins/jetpack/extensions/shared/external-media/index.js

```diff
@@ -43,8 +50,17 @@ if ( isCurrentUserConnected() && 'function' === typeof useBlockEditContext ) {
 		'external-media/replace-media-upload',
 		OriginalComponent => props => {
 			const { name } = useBlockEditContext();
+			const { registerInserterMediaCategory } = useDispatch( 'core/block-editor' );
```

Comment:
[Jetpack supports the WP current version and the immediate previous version](https://github.com/Automattic/jetpack/blob/trunk/docs/coding-guidelines.md#versions-supported). `registerInserterMediaCategory` is only available as of Gutenberg 16.2, so it's not part of WP 6.2 (current WP version) or WP 6.1 (previous WP version). We need to check whether it's available before using it.

Reply:
Now we only register the media categories if the infrastructure for doing so is available.